README.md

```diff
@@ -70,29 +75,30 @@ See the [SingleCounter](https://github.com/elmish/Elmish.WPF/tree/master/src/Sam
 
    The strings identify the binding names to be used in the XAML views. The [Binding module](https://github.com/elmish/Elmish.WPF/blob/master/src/Elmish.WPF/Binding.fs) has many functions to create various types of bindings.
 
-6. Create a WPF user control library project to hold you XAML files, add a reference to this project from your Elmish project, and define your views and bindings in XAML:
+8. Create a WPF user control library project to hold you XAML files, add a reference to this project from your Elmish project, and define your views and bindings in XAML:
 
    ```xaml
    <Window
```

Comment:
The XAML sample in the readme is "optimized" for clarity of example while reading the readme, not renderability. (The reader has already been referred to the SingleCounter sample and told that these are just the central points.) I'd like to keep it the way it was. If you want, feel free to add `(kept simple here for clarity)` or something to that effect before the ending colon on this point.

Reply:
Good point, will change it back